geometric_kernels/spaces/lie_groups.py

```diff
@@ -144,7 +142,15 @@ def __call__(self, gammas):
 
 class MatrixLieGroup(DiscreteSpectrumSpace):
     r"""
-    A base class for Lie groups represented as matrices. The group operation is the matrix multiplication, and the inverse is the matrix inverse.
+    A base class for Lie groups of matrices, subgroups of the general linear
+    group GL(n).
+
+    The group operation is the standard matrix multiplication, and the group
+    inverse is the standard matrix inverse. Despite this, we make the
```

Comment:
"we make the subclasses implement their own multiplication/inverse routines" -> "we defer the implementation of the inverse to the subclasses".

We don't have a special multiplication, do we? Only the inverse is overloaded.

Reply:
Done.